akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala

```diff
@@ -104,7 +104,6 @@ class Slf4jLogger extends Actor with SLF4JLogging with RequiresMessageQueue[Logg
       }
 
     case InitializeLogger(_) =>
-      log.info("Slf4jLogger started")
```

Comment:
I think this should anyway touch the `log` because that is a `lazy val` and if we don't touch it we haven't really started the intiialization yet, and next real log message will do that.

Reply:
How about let `Slf4jLogger` **NOT** to extend `SLF4JLogging`

Reply:
That would be fine, but it would still have to create the logger at some point.

Well, there is a problem with compatibility if removing `SLF4JLogging`, because `Slf4jLogger` is public api and not final. Someone may have extended it and expect that `log`.